app/src/user/user.service.ts

```diff
@@ -506,6 +611,19 @@ export class UserService {
                 },
                 (error: AxiosError) => reject(error)
             );
+            this.httpRequestsService.request(removeUserTechnologies).subscribe(
```

Comment:
Lines from 614 to 626 is an uncontrolled chain of code. Place it into the single chain